airflow/providers/amazon/aws/triggers/emr.py

```diff
@@ -31,6 +32,7 @@
     from airflow.providers.amazon.aws.hooks.base_aws import AwsGenericHook
 
 
+@deprecated(reason="use EmrStepsTrigger instead")
```

Comment:
I don't think we should deprecate it especially if it never work properly just need to fix it so you could move your implementation here